scholia/app/templates/use_usage-over-time.sparql

```diff
@@ -7,8 +7,7 @@ WITH{
   SELECT
     (MIN(?year_) AS ?year) ?work ?use
   WHERE {
-    ?work (wdt:P2283 | wdt:P4510) ?use .
```

Comment:
This was meant to show usage over time, e.g., 
https://scholia.toolforge.org/use/Q41566485#usage-over-time